python/test/test_document_review_checker.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+import tempfile
+import os
+from datetime import datetime
+
+from python.scripts import document_review_checker as check
+
+
+class TestDocumentReviewChecker(unittest.TestCase):
+    def setUp(self) -> None:
+        self.file_path = tempfile.mkdtemp()
+
+    def test_documents_due_for_review(self):
+        with tempfile.NamedTemporaryFile(
+                dir=self.file_path, delete=False, suffix=".md.erb") as file:
+            file.write(b"last_reviewed_on: 2020-03-14")
+            file.close()
+
+        self.addCleanup(os.remove, file.name)
+
+        documents = check.get_documents_due_for_review(self.file_path)
+        self.assertIn(file.name, documents)
+
+    def test_no_documents_to_review(self):
+        today = datetime.today()
+        with tempfile.NamedTemporaryFile(
```

Comment:
lines 26 to 34 could be in a helper function so each test can call it with the date as a function parameter ?

Reply:
# create a temporary file using a context manager
with tempfile.TemporaryFile() as fp:
    fp.write(b'Hello world!')
    fp.seek(0)
    fp.read()
b'Hello world!'
>>>
# file is now closed and removed

Reply:
Now added.